<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHiMDUploadDialog</class>
 <widget class="QDialog" name="QHiMDUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track upload - QHiMDTransfer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/upload_from_md.png</normalon>
   </iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="header">
     <item>
      <widget class="QLabel" name="icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/icons/upload_from_md.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="alltrack_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>please wait while uploading XX track(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="overall">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>overall progress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="AllPBar"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="current" native="true">
     <layout class="QVBoxLayout" name="currentlayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="curtrack_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>current track: trknum - filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="TrkPBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="summary">
     <item>
      <widget class="QLabel" name="success_text">
       <property name="text">
        <string>xx track(s) successfully uploaded</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="failed_text">
       <property name="text">
        <string>xx track(s) could not be uploaded</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonbox">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="details_button">
       <property name="text">
        <string>Show Details</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="details">
     <item>
      <widget class="QTreeWidget" name="ErrorList">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>80</number>
       </attribute>
       <column>
        <property name="text">
         <string>Tracknr.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Error message</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
